src/utils/hub-channel.js

```diff
@@ -22,11 +22,18 @@ const HUB_CREATOR_PERMISSIONS = [
   "update_roles",
   "close_hub",
   "mute_users",
-  "kick_users"
+  "kick_users",
+  "amplify_audio"
 ];
-const VALID_PERMISSIONS =
-  HUB_CREATOR_PERMISSIONS +
-  ["tweet", "spawn_camera", "spawn_drawing", "spawn_and_move_media", "pin_objects", "spawn_emoji", "fly"];
+const VALID_PERMISSIONS = HUB_CREATOR_PERMISSIONS.concat([
+  "tweet",
+  "spawn_camera",
+  "spawn_drawing",
+  "spawn_and_move_media",
+  "pin_objects",
+  "spawn_emoji",
+  "fly"
+]);
```

Comment:
Oh wow, nice catch 😅

Reply:
Yeah at first I was like "Wait, js has an array concatenation operator? And then I realized what was going on... Oh javascript. Funny that it just happened to work mostly correctly by chance.

Reply:
Ah good to see it's fixed, noticed that a year ago and I was just about to push a PR for that!
I wondered too at first if I was missing some obscure syntax  XD